tests/test_sandbox_scripts.py

```diff
@@ -72,36 +72,37 @@ def _sandbox_scripts():
 
 
 @pytest.mark.parametrize("filename", _sandbox_scripts())
-def test_import_succeeds(filename):
+def test_import_succeeds(filename, tmpdir):
```

Comment:
Isolation. There were some sandbox scripts were opening files in the root dir and not cleaning up their mess. I think I actually fixed that, but the extra isolation is probably a good idea anyway.